packages/tooltips/src/styled/StyledTooltip.ts

```diff
@@ -13,23 +13,17 @@ import {
   DEFAULT_THEME,
   getLineHeight
 } from '@zendeskgarden/react-theming';
-
-import { POPPER_PLACEMENT, getArrowPosition } from '../utils/gardenPlacements';
+import { getArrowPosition } from '../utils/gardenPlacements';
+import { ITooltipProps, POPPER_PLACEMENT } from '../utils/types';
 import { StyledParagraph } from './StyledParagraph';
 import { StyledTitle } from './StyledTitle';
 
 const COMPONENT_ID = 'tooltip.tooltip';
 
-export type TOOLTIP_SIZE = 'small' | 'medium' | 'large' | 'extra-large';
-export type TOOLTIP_TYPE = 'light' | 'dark';
-
-export interface IStyledTooltipProps {
-  hasArrow?: boolean;
-  size?: TOOLTIP_SIZE;
+export interface IStyledTooltipProps extends Pick<ITooltipProps, 'hasArrow' | 'size' | 'zIndex'> {
```

Comment:
After some additional noodling, I think this is a fairly decent pattern for public interface reuse within view components.